lib/browser_crawler/reports/csv_report.rb

```diff
@@ -0,0 +1,22 @@
+module BrowserCrawler
+  module Reports
+    # It involves methods which allow to save a store to a csv file
+    class CsvReport
+      def initialize(store:)
+        @store = store
+      end
+
+      def export(save_folder_path:)
+        CSV.open("#{save_folder_path}/crawler_report.csv", 'wb') do |csv|
+          csv << ['pages', 'extracted links']
+          puts @store.inspect
```

Comment:
I think we should remove this 'puts' cause expecting the big object would not make the log readable

Reply:
good catch!